frontend/src/App.js

```diff
@@ -26,53 +29,18 @@ import { StudentProvider } from "./contexts/StudentContext";
 let isLogged = true;
 let isSignedUp = true;
 
-const intialDevSettings = {
-  username: "Staff",
-  userType: "staff",
-  isLogged: true,
-  isSignedUp: true,
-};
-
-const devSettingsReducer = (state, action) => {
-  switch (action.type) {
-    case "set_user_type":
-      return {
-        ...state,
-        userType: action.payload.userType,
-        username: action.payload.username,
-      };
-    case "set_logged":
-      return { ...state, isLogged: action.payload };
-    case "set_sign_up":
-      return { ...state, isSignedUp: action.payload };
-    default:
-      throw new Error();
-  }
-};
-
 // camal case
 const ContentContainer = styled.div`
   width: 100vw;
   padding: 25px 50px;
 `;
 
 function App() {
-  const [state, dispatch] = useReducer(devSettingsReducer, intialDevSettings);
-
-  const handleToggleSettings = e => {
-    dispatch({
-      type: "set_user_type",
-      payload: {
-        userType: e.target.value,
-        username: e.target.value.toUpperCase(),
-      },
-    });
-  };
-
   return (
     <BrowserRouter>
       {isLogged && (
-        <Header userType={state.userType} username={state.username} />
```

Comment:
Need line 75
<Header userType={state.userType} username={state.username} />
for a number of components to access currently logged in user.